Tutorials/Home.md

```diff
@@ -0,0 +1,11 @@
+Welcome to the OPEN-PROM wiki!
```

Comment:
this is not a Wiki anymore, suggested change: refer to the Turorials folder rather than the Wiki